securedrop/tests/conftest.py

```diff
@@ -194,6 +196,12 @@ def alembic_config(config: SDConfig) -> str:
 
 @pytest.fixture(scope='function')
 def source_app(config: SDConfig) -> Generator[Flask, None, None]:
+    config.SOURCE_APP_FLASK_CONFIG_CLS.TESTING = True
+    config.SOURCE_APP_FLASK_CONFIG_CLS.USE_X_SENDFILE = False
+
+    # Disable CSRF checks to make writing tests easier
+    config.SOURCE_APP_FLASK_CONFIG_CLS.WTF_CSRF_ENABLED = False
```

Comment:
This was copied from the config.py created by `make test`. These settings are needed for the tests to pass.